build.sbt

```diff
@@ -9,7 +9,7 @@ lazy val V = new {
   val betterMonadicFor: String = "0.3.0"
   val cats: String             = "1.6.0"
   val catsScalacheck: String   = "0.1.1"
-  val mouse: String            = "0.20"
+  val mouse: String            = "0.21"
```

Comment:
I think this might cause problems, I believe mouse 0.21 is for cats 2.0?

Reply:
You are right. mouse should not be updated